src/main/java/uk/gov/hmcts/reform/pdf/service/exception/InvalidArgumentException.java

```diff
@@ -7,7 +7,8 @@
  * SonarQube reports as error. Max allowed - 5 parents
  */
 @SuppressWarnings("squid:MaximumInheritanceDepth")
-public class InvalidArgumentException extends UnknownErrorCodeException {
+// PMD is complaining about no serial version uid for some reason
+public class InvalidArgumentException extends UnknownErrorCodeException { //NOPMD
```

Comment:
No harm in putting a serial version id for a class which is serializable.

Reply:
Will have a look at it soon, confused on why it wants it for just this one though